ctz/keccak
# Python 3 compatibility

[sweg@CNU352DG8J keccak]$ python2 tests.py
    tests passed
    [sweg@CNU352DG8J keccak]$ python3 tests.py
    tests passed

- use hexlify instead of str.encode('hex')
- use unhexlify and compare actual bytestring values in tests
- bytes2str is now the more aptly-named ilist2bytes, since it converts lists-of-integers into bytestrings
- str2bytes is now the more aptly-named bytes2ilist, since it converts bytestrings into lists-of-integers
- bytestrings prefaced with b where appropriate
- range and xrange shuffled into bilingual compatibility
- use // for integer division
- delicately slice h in some test cases
- parenthesis around print
- newline at end of file

This baby should be ready-to-merge!
